cli/actor.go

```diff
@@ -0,0 +1,409 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli/v2"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/big"
+	"github.com/filecoin-project/lotus/build"
+	"github.com/filecoin-project/lotus/chain/actors"
+	"github.com/filecoin-project/lotus/chain/actors/builtin/miner"
+	types "github.com/filecoin-project/lotus/chain/types"
+
+	"github.com/filecoin-project/specs-actors/actors/builtin"
+	miner0 "github.com/filecoin-project/specs-actors/actors/builtin/miner"
+	builtin2 "github.com/filecoin-project/specs-actors/v2/actors/builtin"
+)
+
+var actorCmd = &cli.Command{
+	Name:  "actor",
+	Usage: "Modify actor info",
+	Subcommands: []*cli.Command{
+		workerCmd,
+		ownerCmd,
+		controlSetCmd,
+		withdrawCmd,
+	},
+}
+
+var workerCmd = &cli.Command{
```

Comment:
We should use the same code for `lotus miner ...` and `lotus-miner actor ...` commands, just with the difference that lotus-miner commands would set a default actor